docs/docs/Support/release-notes.mdx

```diff
@@ -4,6 +4,7 @@ slug: /release-notes
 ---
 
 import Icon from "@site/src/components/icon";
+import McpIcon from '@site/static/logos/mcp-icon.svg';
```

Comment:
Do you still need this if you can import inline? Or is that icon outside the library?